src/main/java/com/rainforestcommerce/rcdb/views/V_ShipmentsRequestViewController.java

```diff
@@ -0,0 +1,80 @@
+package com.rainforestcommerce.rcdb.views;
+import com.rainforestcommerce.rcdb.models.Shipment;
+import com.rainforestcommerce.rcdb.views.ActivityManager.Activity;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableRow;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.layout.VBox;
+import java.io.IOException;
+import java.util.Date;
+
+/**
+ * The controller for the Vendor Shipment Requests View.
+ * Responsible for loading the view's FXML, configuring the view's table
+ * to display Shipment objects, and populating the table with the
+ * Shipment objects retrieved from the database.
+ *
+ * @author Abdullah Al Hamoud <aaa1008@g.rit.edu>
+ *
+ */
+public class V_ShipmentsRequestViewController {
```

Comment:
why shorten `Vendor` here? (the other one is distinct since it's `VendorShipmentDetails`